src/dsp_tools/commands/xmlupload/make_rdf_graph/make_values.py

```diff
@@ -51,157 +69,140 @@ def make_values(properties: list[XMLProperty], res_bnode: BNode, lookups: Lookup
     # To frame the json-ld correctly, we need one property used in the graph. It does not matter which.
     last_prop_name = None
 
-    for prop in properties:
-        single_prop_graph, last_prop_name = _make_one_prop_graph(prop=prop, res_bnode=res_bnode, lookups=lookups)
+    for prop in values:
+        single_prop_graph, last_prop_name = _make_one_prop_graph(val=prop, res_bnode=res_bnode, iri_lookup=lookups)
         properties_graph += single_prop_graph
 
     return properties_graph, last_prop_name
 
 
-def _make_one_prop_graph(prop: XMLProperty, res_bnode: BNode, lookups: Lookups) -> tuple[Graph, URIRef]:
-    prop_name = get_absolute_iri(prop.name, lookups.namespaces)
-    match prop.valtype:
-        case "boolean" | "color" | "decimal" | "geometry" | "geoname" | "integer" | "time" | "uri" as val_type:
-            literal_info = RDF_PROP_TYPE_MAPPER[val_type]
-            transformer = RDF_LITERAL_TRANSFORMER_MAPPER[val_type]
-            properties_graph = _make_simple_prop_graph(
-                prop=prop,
+def _make_one_prop_graph(val: IntermediaryValue, res_bnode: BNode, iri_lookup: IRILookup) -> tuple[Graph, URIRef]:
```

Comment:
Does this produce 1 prop graph or 1 value graph?

Reply:
It creates one graph with one property which also corresponds to one value. I can rename it if it is confusing .